arches_for_science/apps.py

```diff
@@ -0,0 +1,7 @@
+from django.apps import AppConfig
+
+
+class ArchesForScienceConfig(AppConfig):
+    name = "arches_for_science"
+    verbose_name = "Arches_for_science"
```

Comment:
This is the human readable name, so we can:
```suggestion
    verbose_name = "Arches for Science"
```